test/Microsoft.DotNet.Darc.Tests/DependencyFileManagerTests.cs

```diff
@@ -65,7 +65,7 @@ public class DependencyFileManagerTests
         "https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-4ac4c036/nuget/v3/index.json" })]
     public async Task UpdatePackageSourcesTests(string testName, string[] managedFeeds)
     {
-        DependencyFileManager dependencyFileManager = new DependencyFileManager(null, new VersionDetailsParser(), NullLogger.Instance);
+        DependencyFileManager dependencyFileManager = new DependencyFileManager((IGitRepo)null, new VersionDetailsParser(), NullLogger.Instance);
```

Comment:
It would be cleaner to define a DependencyFileManager constructor overload without the first parameter so we don't have to pass in a (cast) null.